node/tests/GlideClusterClient.test.ts

```diff
@@ -68,9 +69,19 @@ describe("GlideClusterClient", () => {
         cluster = clusterAddresses
             ? await ValkeyCluster.initFromExistingCluster(
                   parseEndpoints(clusterAddresses),
+                  async (addresses: [string, number][]) => {
+                      return getServerVersion(addresses, true);
+                  },
               )
             : // setting replicaCount to 1 to facilitate tests routed to replicas
-              await ValkeyCluster.createCluster(true, 3, 1);
+              await ValkeyCluster.createCluster(
+                  true,
+                  3,
+                  1,
+                  async (addresses: [string, number][]) => {
+                      return getServerVersion(addresses, true);
+                  },
```

Comment:
```suggestion
                  getServerVersion
```
once you can properly pass cluster mode as a second arg to the callback, you can simplify that

Reply:
updated.